db/migrate/20180120082635_lead_belongs_to_account.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class LeadBelongsToAccount < ActiveRecord::Migration[5.1]
+  def change
+    add_column :leads, :account_id, :int
+
+    add_index :leads, :account_id
```

Comment:
Migration needs to migrate the values from the Account -> Relationship. We should also add a second migration to remove the old relationship.